boggle.js

```diff
@@ -1,3 +1,22 @@
+// import { strict } from "assert";
+let process = require('process');
+const fs = require('fs');
+let dictionary = process.argv[2];
+let dict = {};
+var text = fs.readFileSync(dictionary, 'utf-8');
+let marked = [
+
+  ['_', '_', '_', '_'],
+  ['_', '_', '_', '_'],
+  ['_', '_', '_', '_'],
+  ['_', '_', '_', '_'],
+];
+let string = text.toUpperCase().split(/\s*\b\s*/);
```

Comment:
Is this to remove trailing whitespace? You can use `String.prototype.trimEnd` if so.